Automattic/jetpack
# Subscribe block: email field required in WPCOM

This fixes a regression possibly introduced in https://github.com/Automattic/jetpack/pull/26947

* Make sure the email field is required in the subscribe block

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

* build, sync to wpcom
*  sandbox bestmemberships or another wpcom site that has a subscribe block
* Delete the prefilled email, making sure its empty
* Try to submit the form
* It should not let you


![Zrzut ekranu 2023-02-16 o 12 40 39](https://user-images.githubusercontent.com/3775068/219356447-c4adfe24-c9ea-4d62-a26e-b8a3e0af748a.png)
